app/config/variables.php

```diff
@@ -420,12 +420,93 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_STORAGE_DEVICE',
+                'description' => 'Select default storage device. The default value is \'Local\'. List of supported adapters are \'Local\', \'S3\' and \'DOSpaces\'.',
+                'introduction' => '0.13.0',
+                'default' => 'Local',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_ACCESS_KEY',
+                'description' => 'AWS S3 storage access key. Required when the storage adapter is set to S3. You can get your access key from your AWS console',
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_SECRET',
```

Comment:
```suggestion
                'name' => '_APP_STORAGE_S3_SECRET',
```
Similar comment as above